ipfs-cluster-ctl/formatters.go

```diff
@@ -82,23 +83,41 @@ func textFormatPrintIDSerial(obj *api.IDSerial) {
 	}
 
 	fmt.Printf("%s | %d peers\n", obj.ID, len(obj.ClusterPeers))
-	fmt.Println("  > Addresses:")
+	addrs := sort.StringSlice{}
```

Comment:
I'm going to do addrs := make(sort.StringSlice, 0, len(obj.Addresses)).

The problem is not the append per se, but the slice capacity which forces appends to re-allocate.